CONTRIBUTING.md

```diff
@@ -30,4 +30,19 @@ This project uses ZenHub for tracking of issues and roadmap.
 
 ## Reporting an Issue
 
-To report an issue, or to suggest an idea for a change, open an [issue](../../issues/new). It is best to check our existing [issues](../../issues) first to see if a similar one has already been opened and discussed.
\ No newline at end of file
+To report an issue, or to suggest an idea for a change, open an [issue](../../issues/new). It is best to check our existing [issues](../../issues) first to see if a similar one has already been opened and discussed.
+## Getting Feedback on Your Contribution
+
+When creating a PR for one of our projects it is important to get your work reviewed by other members of the team. Below are some steps to getting your work reviewed. <br>
+* Assign a Reviewer from the list of reviewers on the project.
+* Move the linked issue(s) from the `In Progress` column to `Ready For Review` column on the [Project Codeflare Sprint Board](https://github.com/orgs/project-codeflare/projects/8/views/3).
```

Comment:
Nitpick 
This assumes the contributor is  using view 3, which you've linked. The default view is view 1 though, which doesn't have status columns. 

I'd suggest simply:
```suggestion
* Change the status of the issue from `In Progress` to `Ready For Review` on the [Project Codeflare Sprint Board](https://github.com/orgs/project-codeflare/projects/8/views/3).
```

It makes sense then for any view

Reply:
Done & changed the wording for the `In Review` and `Closed` statuses